.github/workflows/test.yml

```diff
@@ -41,29 +41,38 @@ jobs:
       with:
         install_url: https://releases.nixos.org/nix/nix-2.18.8/install
         nix_path: nixpkgs=channel:${{ env.CURRENT_STABLE_CHANNEL }}
-    - name: Install ${{ env.CURRENT_STABLE_CHANNEL }} channel
+    - name: Install channels
       run: |
+        nix-channel --add https://github.com/LnL7/nix-darwin/archive/master.tar.gz darwin
         nix-channel --add https://nixos.org/channels/${{ env.CURRENT_STABLE_CHANNEL }} nixpkgs
         nix-channel --update
-    - name: Install nix-darwin and test
+    - name: Install nix-darwin
       run: |
         export NIX_PATH=$HOME/.nix-defexpr/channels
 
-        # We run nix-darwin twice to test that it can create darwin-configuration correctly for us
-        # but we expect it to fail setting up /etc/nix/nix.conf
-        nix-shell -A installer || true
+        mkdir -p ~/.config/nix-darwin
+        cp modules/examples/simple.nix ~/.config/nix-darwin/configuration.nix
 
         nixConfHash=$(shasum -a 256 /etc/nix/nix.conf | cut -d ' ' -f 1)
         /usr/bin/sed -i.bak \
           "s/# programs.fish.enable = true;/nix.settings.access-tokens = [ \"github.com=\${{ secrets.GITHUB_TOKEN }}\" ]; environment.etc.\"nix\/nix.conf\".knownSha256Hashes = [ \"$nixConfHash\" ];/" \
-          ~/.nixpkgs/darwin-configuration.nix
+          ~/.config/nix-darwin/configuration.nix
 
-        nix-shell -A installer
-        nix-shell -A installer.check
-    - name: Build and activate default derivation
+        nix run .#darwin-rebuild \
+          --extra-experimental-features "nix-command flakes" \
+          -- switch \
+          -I darwin-config=$HOME/.config/nix-darwin/configuration.nix
+    - name: Switch to new configuration
       run: |
         . /etc/static/bashrc
```

Comment:
not actually a change made in this PR, but is there a reason this is sourcing `/etc/static/bashrc` instead of `/etc/bashrc`?